src/App.test.js

```diff
@@ -2,8 +2,8 @@ import React from 'react'
 import { render } from '@testing-library/react'
 import App from './App'
 
-// test('renders learn react link', () => {
-//   const { getByText } = render(<App />)
-//   const linkElement = getByText(/learn react/i)
-//   expect(linkElement).toBeInTheDocument()
-// })
+test('renders learn react link', () => {
+  const { getByText } = render(<App />)
+  const linkElement = getByText(/learn react/i)
+  expect(linkElement).toBeInTheDocument()
```

Comment:
This test fails, because there is no element with text learn react.. maybe you forgot to delete this test?